### DISCUSSION TITLE
Adding Permissions Policy Headers to Play Application server responses

### DISCUSSION THREAD

**Discussion by teagy** [playframework/playframework#12937] [Q&A]
Hi all I know that the Play framework has built in configuration support for CSP directives but has there been any support added for the Permissions Policy headers? I would like to add these to our application server responses.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Permissions_Policy

**Comment by mkurz** [comment]
There is no built-in support like we have for CSP, however if the goal is to send a ` Permissions-Policy` header with each response, you can do so by using a filter: https://www.playframework.com/documentation/3.0.x/Filters
(The CSP headers are added via a filter as well)
